src/components/CountriesList/CountriesList.tsx

```diff
@@ -1,23 +1,31 @@
-import { StyledDisplaySection } from "./CountriesList.styles";
+import { StyledDisplaySection, StyledH4 } from "./CountriesList.styles";
 import { CountryCard } from "../countryCard/CountryCard";
 import { useCountriesContext } from "../../context/CountriesDataContext";
 import { CountriesProps } from "./CountriesProps";
 
 export const CountriesList = () => {
-	const { error, countries, selectCountryRegion } = useCountriesContext();
+	const { inputValue, error, countries, selectCountryRegion } =
+		useCountriesContext();
+
+	const filteredData = countries
```

Comment:
switch to two separate functions and do not chain .filters()

Reply:
fixed.